plug/pip_resolve.py

```diff
@@ -114,13 +115,25 @@ def create_dependencies_tree_by_req_file_path(requirements_file_path):
         installed = [p for p in dist_index]
         for r in required:
             if r.lower() not in installed:
-                sys.exit('Required package missing: ' + r.lower())
+                msg = 'Required package missing: ' + r.lower()
```

Comment:
Want to add `SNYKSNYKSNYK` here or similar? We need to identify our own code's output inside another code's output?